inc/endpoints/class-components-endpoint.php

```diff
@@ -453,26 +473,15 @@ public function add_api_link_to_posts( array $actions, \WP_Post $post ) : array
 		// Get post permalink.
 		$permalink = get_permalink( $post );
 
-		// Extract path.
-		$path = wp_parse_url( $permalink, PHP_URL_PATH );
-
-		// Apply path to rest URL for Irving components endpoint.
-		// Use site_url() instead of rest_url() because WP-Irving uses WP_HOME
-		// as the Irving App, and rest_url() uses WP_HOME instead of
-		// WP_SITEURL, which is what we need to use.
-		$path_url = add_query_arg(
-			'path',
-			$path,
-			site_url( rest_get_url_prefix() . '/irving/v1/components' )
-		);
-
-		$path_url = apply_filters( 'wp_irving_post_row_action_path_url', $path_url, $post );
+		// Get the API URL, allowing it to be filtered.
+		$api_url = self::get_wp_irving_api_url( $permalink );
```

Comment:
Since you're actually getting the API URL for a specific path, I'd leave the variable named `$path_url` for clarity.